test/libsolidity/StandardCompiler.cpp

```diff
@@ -1196,7 +1196,7 @@ BOOST_AUTO_TEST_CASE(evm_version)
 	BOOST_CHECK(result["contracts"]["fileA"]["A"]["metadata"].get<std::string>().find("\"evmVersion\":\"cancun\"") != std::string::npos);
 	// test default
```

Comment:
You should also add a check for `prague` above.

I'd just rewrite it as a loop, using the recently added `EVMVersion::allVersions()`.

Reply:
Refactored as suggested.